merge/schema_change_test.go

```diff
@@ -178,75 +178,3 @@ func TestGranularToAtomicSchemaChanges(t *testing.T) {
 		})
 	}
 }
-
-func TestAtomicToGranularSchemaChanges(t *testing.T) {
```

Comment:
Can we actually keep the test and update it with the new behavior? If we document what happens when someone goes from atomic to granular, then we should have a test to confirm that behavior.

Reply:
Seems useful to me as well. Might also be worth adding a comment in the tests explaining the behavior further.

Reply:
done, let me know if the comments are sufficient